operational_analysis/methods/electrical_losses.py

```diff
@@ -28,16 +36,35 @@ class ElectricalLosses(object):
     """ 
 
     @logged_method_call
-    def __init__(self, plant, correction_thresh=.95):
+    def __init__(self, plant, UQ = 'Y', num_sim = 20000, uncertainty_meter=0.005, uncertainty_scada=0.005,
+                 uncertainty_correction_thresh=(0.9,0.995), correction_thresh=.95):
         """
         Initialize electrical losses class with input parameters
 
         Args:
          plant(:obj:`PlantData object`): PlantData object from which EYAGapAnalysis should draw data.
-
+         num_sim:(:obj:`int`): number of Monte Carlo simulations
+         UQ:(:obj:`str`): choice whether to perform ('Y') or not ('N') uncertainty quantification
```

Comment:
Can you change this to a boolean - e.g., True is UQ?

Reply:
Changed!